src/network/networkprotocol.h

```diff
@@ -942,16 +942,16 @@ enum NetProtoCompressionMode {
 };
 
 const static std::string accessDeniedStrings[SERVER_ACCESSDENIED_MAX] = {
-	"Invalid password",
-	"Your client sent something the server didn't expect.  Try reconnecting or updating your client",
-	"The server is running in simple singleplayer mode.  You cannot connect.",
-	"Your client's version is not supported.\nPlease contact server administrator.",
-	"Player name contains disallowed characters.",
-	"Player name not allowed.",
-	"Too many users.",
-	"Empty passwords are disallowed.  Set a password and try again.",
-	"Another client is connected with this name.  If your client closed unexpectedly, try again in a minute.",
-	"Server authention failed.  This is likely a server error."
+	"You have entered a wrong password.",
```

Comment:
I don't think `wrong` is gramatically correct here.  "The password you entered is wrong." would be better, but I'd just use `incorrect` or `invalid`.